lib/patternfly/patternfly-4-overrides.scss

```diff
@@ -0,0 +1,53 @@
+/*** PF4 overrides ***/
+
+/* WORKAROUND: Override word-break bug */
+/* See: https://github.com/patternfly/patternfly-next/issues/2325 */
+.pf-c-table td {
+    word-break: normal;
+    overflow-wrap: break-word;
+}
+
+/* WORKAROUND:  Dropdown (PF4): Caret is not properly aligned bug */
+/* See: https://github.com/patternfly/patternfly/issues/2715 */
+/* Align the icons inside of all dropdown toggles. */
+/* Part 1 of 2 */
+.pf-c-dropdown__toggle-button {
+  display: flex;
+  align-items: center;
+}
+
+/* Make split button dropdowns the same height as their sibling. */
+/* Part 2 of 2 */
+.pf-m-split-button {
+  align-items: stretch;
+}
+
+/* WORKAROUND: Navigation problems with Tertiary Nav widget on mobile */
+/* See: https://github.com/patternfly/patternfly-design/issues/840 */
+/* Helper mod to wrap pf-c-nav__tertiary */
+.ct-m-nav__tertiary-wrap {
+    .pf-c-nav__list {
+        flex-wrap: wrap;
+    }
+
+    .pf-c-nav__scroll-button {
+        display: none;
+    }
+}
+
+/* Helper mod to center pf-c-nav__tertiary when it wraps */
+.ct-m-nav__tertiary-center {
+    .pf-c-nav__list {
+        justify-content: center;
+    }
+}
+
+ul.pf-c-select__menu {
+    max-height: 20rem;
+    overflow-y: scroll;
+}
+
+/* Drop-down menu should overlay other elements */
```

Comment:
This was added by me now, without it, when one has more than two pods and tries to select the dropdown in the first pod, it gets hidden by the next pod action buttons.